lib/node_modules/@stdlib/fs/append-file/benchmark/benchmark.js

```diff
@@ -0,0 +1,82 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var join = require( 'path' ).join;
+var bench = require( '@stdlib/bench' );
+var readFile = require( '@stdlib/fs/read-file' ).sync;
+var pkg = require( '@stdlib/fs/append-file/package.json' ).name;
+var appendFile = require( './../lib' );
+
+
+// FIXTURES //
+
+var FILEPATH = join( __dirname, 'fixtures', 'file.txt' );
+var DATA = readFile( FILEPATH );
+
+
+// MAIN //
+
+bench( pkg, function benchmark( b ) {
+	var i;
+
+	i = 0;
+	b.tic();
+
+	return next();
+
+	function next() {
+		i += 1;
+		if ( i <= b.iterations ) {
+			return appendFile( FILEPATH, DATA, done );
```

Comment:
Wait...are you repeatedly appending the file over and over again? If I run this benchmark multiple times, this could lead to a huge file. You need to create a fresh `tmp` file for each invocation of `bench`, but before `t.tic()` and you need to delete that tmp file after `t.toc()`. Otherwise, this could lead to a massive amount of used resources.